wgpu-core/src/command/mod.rs

```diff
@@ -361,19 +361,19 @@ pub enum PassErrorScope {
     #[error("In a set_scissor_rect command")]
     SetScissorRect,
     #[error("In a draw command")]
-    Draw,
+    Draw(Option<id::RenderPipelineId>),
```

Comment:
Can we reduce the duplication here? Something like:
```rust
Draw {
  indexed: bool,
  indirect: bool,
  pipeline: Option<id::RenderPipelineId>,
},
```

Reply:
hmm, that would make the type ever so slightly larger ;) 
I'd prefer to punt it to later

Reply:
Does it matter how large the error type is? It's not like you'd be juggling it in perf-sensitive contexts

Reply:
No, that was in jest :)